packages/web-components/package.json

```
@@ -96,8 +96,8 @@
   },
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
-    "@microsoft/fast-element": "^1.0.0",
-    "@microsoft/fast-foundation": "1.16.0",
+    "@microsoft/fast-element": "1.1.0",
+    "@microsoft/fast-foundation": "1.21.0",
```

I think we want the `fast-element` instance to stay on the major version. The `fast-foundation` version can get bumped but let's keep it opted into semver:

```suggestion
    "@microsoft/fast-foundation": "^1.21.0",
```

---

Ugh, that change was in a different branch. Not sure why it was included.

---

Pulled this change out.